compiler/src/iree/compiler/Codegen/Common/TileAndDistributeToWorkgroupsPass.cpp

```diff
@@ -448,12 +450,14 @@ void TileAndDistributeToWorkgroupsPass::runOnOperation() {
     });
 
     {
+      SmallVector<int64_t> staticNumWorkgroup = getNumWorkgroup(funcOp);
```

Comment:
Trying to understand why this needs to be here (I know it is passed into the pattern, but cant quite place the reason why it is passed in now as opposed to this not being the case earlier.)

Reply:
I could get it directly in the pattern but that means I need to do it for every affineMin and I would have to look for the exportOp parent of the AffineMinOp. Did you have another solution?

Reply:
I am more concerned about exposing the `getNumWorkgroups` function. It shouldnt be something to be relied upon.... Things that rely on it are not sound IMO. If this is a transitionary thing then its fine. Maybe rename it to `getStaticNumWorkgroups`.